### PR TITLE
GH-01-ListOfCryptoCurrencies

### PR DESCRIPTION
**Pull request by GonzaloHz**
image
screen actualiced

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {SafeAreaView} from 'react-native';
+import HomeScreen from './src/screens/HomeScreen/HomeScreen';
+
+const App = () => {
+  return (
```
Comment (tomasdematos) [inline App.tsx]: use implicit return

File: src/components/CryptoItem/CryptoItem.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+const CryptoItem = ({
+  symbol,
+  name,
+  value,
+  percentage,
+  signal,
+  slug,
+  image,
+}: {
```
Comment (tomasdematos) [inline src/components/CryptoItem/CryptoItem.tsx]: create a type for this object

Comment (tomasdematos) [reply src/components/CryptoItem/CryptoItem.tsx]: actualy, the type should be DataTypes, change the name for a more descriptive one, like CryptoTypes

File: src/components/CryptoItem/CryptoItem.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <Text>{symbol}</Text>
+    </ItemLeftContainer>
+    <ItemRigthContainer>
+      <ItemTitle>${value}</ItemTitle>
+      <ItemSubSubContainer>
+        <StyledArrow
+          source={signal ? ArrowUp : ArrowDown}
+          alt="image not found"
+        />
+        <TextPercentage color={signal ? 'green' : 'red'}>
```
Comment (tomasdematos) [inline src/components/CryptoItem/CryptoItem.tsx]: the validation should be inside the component: for example: 

<TextPercentage positive={signal}

File: src/components/CryptoItem/style.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components"
+
+export const StyledImage = styled.Image`
+  max-height: 60px;
+  max-width: 60px;
+  margin-left: 20px;
```
Comment (tomasdematos) [inline src/components/CryptoItem/style.ts]: margin-horizontal: 20px

File: src/components/CryptoItem/style.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+export const StyledArrow = styled.Image`
+  max-height: 20px;
+  max-width: 20px;
+`;
+
+export const ItemContainer = styled.View`
+  display: flex;
+  flex-direction: row;
+  padding: 10px 0px;
+  border-color: gray;
```
Comment (tomasdematos) [inline src/components/CryptoItem/style.ts]: create and use a theme file: 

src/utils/theme

```
const theme = {
   colors: {
      white: "#ffffff",
      black.....
   }
}
export default theme;
```

File: src/components/CryptoItem/style.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+export const ItemSubSubContainer = styled.View`
+  display: flex;
+  flex-direction: row;
+  margin-left: auto;
+`;
+
+export const ItemTitle = styled.Text`
+  font-weight: bold;
+  color: black;
```
Comment (tomasdematos) [inline src/components/CryptoItem/style.ts]: theme

File: src/containers/CurrencyList/CurrencyList.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+      symbol={item.symbol}
+      name={item.name}
+      signal={item.signal}
+      percentage={item.percentage}
+      value={item.value}
+      slug={item.slug}
+      image={item.image}
+    />
+  );
+  return (
```
Comment (tomasdematos) [inline src/containers/CurrencyList/CurrencyList.tsx:8]: add a space before the return

File: src/containers/CurrencyList/CurrencyList.tsx [hunk 8]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {SafeAreaView, FlatList} from 'react-native';
+import CryptoItem from '../../components/CryptoItem/CryptoItem';
+
+const CurrencyList = ({data}) => {
+  const renderItem = ({item}) => (
+    <CryptoItem
```
Comment (tomasdematos) [inline src/containers/CurrencyList/CurrencyList.tsx]: why you don't send directly the item ?, insted of a prop for every part of it

File: src/containers/CurrencyList/style.ts [hunk 9]
```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components"
+
```
Comment (tomasdematos) [inline src/containers/CurrencyList/style.ts]: the styles in this file are repeated

File: src/screens/HomeScreen/HomeScreen.tsx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+    symbol: 'XRP',
+    value: '0.220568',
+    percentage: '2,47',
+    signal: false,
+    image: xprLogo,
+  }
+]
+
+const HomeScreen = () => {
+  return (
```
Comment (tomasdematos) [inline src/screens/HomeScreen/HomeScreen.tsx]: use implicit return

File: src/screens/HomeScreen/style.ts [hunk 11]
```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+export const StyledHeader = styled.View`
+  width: 100%;
+  height: 120px;
+  background-color: #385775;
```
Comment (tomasdematos) [inline src/screens/HomeScreen/style.ts]: theme

File: src/screens/HomeScreen/style.ts [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+  display: flex;
+  flex-direction: row;
+`;
+
+export const StyledView = styled.View`
+  width: 100%;
+  min-height: 200px;
+  background-color: #ffffff;
```
Comment (tomasdematos) [inline src/screens/HomeScreen/style.ts]: theme

File: src/screens/HomeScreen/style.ts [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+
+export const StyledText = styled.Text`
+  font-weight: bold;
+  color: white;
+  padding-top: 10%; 
+  padding-left: 5%;
+  font-size: 22px;
+`;
+
+export const StyledImage = styled.Image`
```
Comment (tomasdematos) [inline src/screens/HomeScreen/style.ts]: it is redundant to call all components StyledXX

File: App.tsx (hunk 1 above)
Comment (VicenBayugar) [inline App.tsx]: use implicit return

**Pull request merged**